medusa/restore_cluster.py

```diff
@@ -34,17 +34,17 @@
 def orchestrate(config, backup_name, seed_target, temp_dir, host_list, keep_auth, bypass_checks,
                 verify, keyspaces, tables, pssh_pool_size, use_sstableloader=False):
     monitoring = Monitoring(config=config.monitoring)
+    retained_seed_target = seed_target
     try:
         restore_start_time = datetime.datetime.now()
-        if seed_target is not None:
+        if seed_target is not None and host_list is None:
             keep_auth = False
 
         if seed_target is None and host_list is None:
-            err_msg = 'You must either provide a seed target or a list of host'
-            logging.error(err_msg)
-            raise Exception(err_msg)
+            retained_seed_target = socket.getfqdn()
```

Comment:
Why using retained_seed_target, if in that case seed_target is empty, why not just setting it directly instead of creating a new variable for the same thing?

Reply:
you're right, that part is fairly unclear...

The checks were done to identify when a remote restore was done to force the keep_auth value, but that's not relevant to do it this way as we can do remote restores without specifying a host list file.
I'll rework all this and ask the use if he wants to retain `system_auth` unless `--keep-auth` is set explicitly.